.github/workflows/e2e-short.yaml

```diff
@@ -1,17 +1,10 @@
-name: Run e2e tests
+name: Run short e2e tests
 
 on:
-  push:
-    branches:
-      - "main"
   pull_request:
 
-env:
-  RANCHER_PASSWORD: ${{ secrets.RANCHER_PASSWORD }}
-
 jobs:
   e2e:
-    concurrency: test-e2e
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
```

Comment:
We need to add `if: always()` to the run artifacts collection, so we can have something to debug.